public/js/pimcore/helpers.js

```diff
@@ -1077,10 +1078,10 @@ pimcore.helpers.uploadDialog = function (url, filename, success, failure, descri
                 win.show();
 
                 let finishedErrorHandler = function (e) {
-                    this.activeUploads--;
+                    activeUploads--;
```

Comment:
do we really need a global global variable here? Or do I miss something?

Reply:
@fashxp The variable is defined here: https://github.com/pimcore/admin-ui-classic-bundle/pull/501/files#diff-efc7c8f1d31756b699ed50c0214db7ca6f13bf9544340e02d33e64624a5d3b84R1089